package.json

```diff
@@ -44,8 +44,9 @@
     "bluebird": "^3.5.0",
     "byline": "^5.0.0",
     "chalk": "^1.1.3",
-    "clay-utils": "^1.1.1",
+    "clay-utils": "^1.3.0",
     "console-log-level": "^1.4.0",
+    "deep-equal": "^1.0.1",
```

Comment:
any reason why you're using this instead of `lodash/_.isEqual`?

Reply:
Good idea, will change.